src/game.cpp

```diff
@@ -3725,7 +3725,8 @@ std::vector<std::string> game::list_active_characters()
  */
 void game::write_memorial_file(std::string sLastWords)
 {
-    const std::string &memorial_dir = FILENAMES["memorialdir"];
+    const std::string &memorial_dir = FILENAMES["memorialdir"] +  world_generator->active_world->world_name + "/";
```

Comment:
That would look to be a good sign. FWIW I think the C API is the unchanged but perhaps someone more informed can enlighten us?